pages/content/reports/2024/index.md

```diff
@@ -0,0 +1,232 @@
+
+Special shout outs to volunteer reviewer [Josh-Cena](https://github.com/Josh-Cena), who alone took on 18.3% of reviews in 2024 for mdn/content, and to [Daniel D. Beck](https://github.com/ddbeck) who reviewed more than half (56.4%) of all web-features pull requests in 2024. A pleasure to work with both of you and your impact in 2024 on these two projects is very noteworthy! Thank you!
+
+Here's the summary of reviewed PRs:
+
+| Project                 | Total reviewed PRs | OWD reviewed PRs |
+| ----------------------- | ------------------ | ---------------- |
+| mdn/content             | 4595               | 1254 (27.3%)     |
+| mdn/browser-compat-data | 2209               | 1154 (52.3%)     |
+| webdx/web-features      | 1961               | 135 (6.9%)       |
+
+## Sovereign Tech Fund
+
+Open Web Docs is very proud to have received a second round investment from the Sovereign Tech Fund at the beginning of 2024. The [Sovereign Tech Fund (STF)](https://www.sovereign.tech/programs/fund) supports the development, improvement, and maintenance of open digital infrastructure in the public interest. The [Sovereign Tech Fund’s Contribute Back Challenges](https://www.sovereign.tech/programs/challenges) selected Open Web Docs as a partner to implement improved tooling and maintenance of the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) and further development of the [openwebdocs/mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) tool.
+
+As part of this program, that continued from the first round in 2023, we completed key milestones:
+
+- Open Web Docs (OWD) automated browser compatibility data updates using OWD’s [mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) project. Now, browser compatibility information is available to web developers for new browser releases faster than ever.
+- OWD automatically detects added and removed browser features for 90.15% of the entire web platform when a new browser is released. The data is recorded in the open-source [BCD](https://github.com/mdn/browser-compat-data) project, which is used by tools commonly used by web developers, such as [CanIUse, MDN Web Docs, and used by](https://caniuse.com/) [Baseline](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility) and [web-features](https://github.com/web-platform-dx/web-features).
+- OWD runs tests on mobile browsers and WebViews to have accurate compatibility data for these platforms and to point out differences between mobile and desktop browsers.
+- We also made our tooling more generally available, so that new tools, such as [runtime-compat-data](https://runtime-compat.unjs.io/) can display compatibility across different JavaScript runtimes. We hope that our work on this data will contribute to better understanding of compatibility issues, to more interoperability, and better standardization on the web platform.
```

Comment:
```suggestion
- We also made our tooling more generally available, so that new tools, such as [runtime-compat-data](https://runtime-compat.unjs.io/) can display compatibility across different JavaScript runtimes. We are proud that our work on this data contributes to better understanding of compatibility issues, to more interoperability, and better standardization on the web platform.
```